docs/src/checklists.md

```diff
@@ -0,0 +1,31 @@
+# Checklists
+
+The purpose of this page is to collate a series of checklists for commonly
+performed changes to the source code of Documenter.
+
+In each case, copy the checklist into the description of the pull request.
+
+## Making a release
+
+In preparation for a release, use the following checklist.
+
+````
+## Pre-release
+
+ - [ ] Change the version number in `Project.toml`
+        * If the release is breaking, increment MAJOR
+        * If the release adds a new user-visible feature, increment MINOR
+        * Otherwise (bug-fixes, documentation improvements), increment PATCH
+ - [ ] Update `CHANGELOG.md`, following the existing style
+ - [ ] Check that the commit messages in this PR do not contain `[ci skip]`
+
+## The release
+
+ - [ ] After merging this pull request, comment `[at]JuliaRegistrator register`
```

Comment:
I've recently started using JuliaHub's registration UI for tagging. We could document both ways?

Reply:
I don't know how that works